### ISSUE TITLE
Bad visibility of search field after first search

### ISSUE THREAD

**Issue by johannesgunnar** [godotengine/godot-docs#2778]
Because of the vast size of the Godot docs when using the online docs (readthedocs) it becomes increasingly difficult to search a second time as the content in the sidebar is so long you end up spending 10+ seconds scrolling with your mouse to reach the search field again. Page up/page down/home/etc keys do nothing as they stop working when you reach top of the content area.

See video

![before](https://user-images.githubusercontent.com/2924543/65387741-c1742700-dd39-11e9-9da8-f271feb940d2.gif)

If we have the power to do so in readthedocs I would recommend something like "position: fixed" on the search field so that it always stays on top. Something like showcased in the following clip. (although a better implementation would probably be where Godot logo would scroll out of view only leaving the search field visible)

![after](https://user-images.githubusercontent.com/2924543/65387745-cdf87f80-dd39-11e9-998d-369e6979a18d.gif)

**Comment by Calinou** [comment]
> If we have the power to do so in readthedocs I would recommend something like "position: fixed" on the search field so that it always stays on top. Something like showcased in the following clip. (although a better implementation would probably be where Godot logo would scroll out of view only leaving the search field visible)

This is doable with custom CSS and JavaScript. I might have a look in the future once #2935 is merged.

**Comment by golddotasksquestions** [comment]
Maybe only have the search bar locked when scolling down? The Logo takes up a lot of real estate if all I want to see is the result (which is indicated by scolling down)

**Comment by golddotasksquestions** [comment]
I just realized having to scroll all the way up is the reason I never use the build in online search in the docs webpage but always go to an online search engine and type "Godot some_method()", because it's so much faster.

![no_searchbar_docs](https://user-images.githubusercontent.com/47016402/70989850-a90b7280-20c4-11ea-8505-0b398a7fffce.gif)

If the search bar would always stick in the top left corner, I would definitely use it more, if not exclusively.

**Comment by RabbitB** [comment]
As @Calinou suggested, this is doable as custom CSS. As a stop-gap measure until something official is implemented, I updated my custom Stylish theme with a screen-fixed search-field. I also implemented a scroll-bar for the left-menu, as it was really annoying me that I couldn't scroll through it easily without also scrolling the main content.

The caveat to this stop-gap is that I added it to my personally maintained custom theme (it's a lot of work to create a new theme just to implement these changes only, and now maintain both), so I hope you like dark themes.

[Dark Godot Docs](https://userstyles.org/styles/155788/dark-godot-docs)

Here's a preview of what it looks like (sorry, no video):
![2020-01-09 15 43 53 docs godotengine org 386de770ab7c](https://user-images.githubusercontent.com/6007524/72114646-19d73080-32f9-11ea-8b1f-d7d7fdee5406.png)
